documentation/tutorials/check.md

```diff
@@ -0,0 +1,174 @@
+---
+layout: documentation
+menu: check
+subtitle: 'Tutorial: Using Check with SQL Server'
+---
+# Tutorial: Using Flyway Check with SQL Server
+
+{% include enterprise.html %}
+
+## Introduction
+
+This is an enteprise feature, but the beta version is available to Teams users.
+
+The new `check` command is run prior to migrating, to give us confidence in the state of the post-migration schema by producing a report. 
+
+One or more of the following flags must be set, which determine what the report contains:
+
+ - `-changes` produces a report of all the changes that will be applied to the schema in the next migration. 
+ - `-drift` produces a report showing objects in the schema which are not the result of any of the currently applied migrations, i.e. previous changes made outside of Flyway.
+
+More information can be found [here](/documentation/command/check).
```

Comment:
Instead of `here` could it be `on the command page` (gives people some notion of where the link will take them)